src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/ReferralServiceTest.kt

```diff
@@ -286,6 +287,19 @@ class ReferralServiceTest @Autowired constructor(
     assertThat(referralService.getSentReferral(sentReferral.id)).isNotNull()
   }
 
+  @Test
+  fun `sending a draft referral generates a human-readable reference for it`() {
+    val user = AuthUser("user_id", "auth_source", "user_name")
+    val draftReferral = referralService.createDraftReferral(user, "X123456", sampleIntervention.id)
+
+    val category = draftReferral.intervention.dynamicFrameworkContract.serviceCategory.name
+    Mockito.`when`(referenceGenerator.generate(category)).thenReturn("AA1234ZZ")
```

Comment:
the mockito kotlin extension provides classes where these backtick method calls aren't nessecary 

e.g `whenever(referenceGenerator...`

Reply:
done in a1f4b893c70b733a4ac3b69a33c41605e62384c5